doc/sphinx-guides/source/installation/config.rst

```diff
@@ -2517,6 +2517,13 @@ Each set of labels is identified by a curationLabelSet name and a JSON Array of
 
 ``curl -X PUT -d '{"Standard Process":["Author contacted", "Privacy Review", "Awaiting paper publication", "Final Approval"], "Alternate Process":["State 1","State 2","State 3"]}' http://localhost:8080/api/admin/settings/:AllowedCurationLabels``
 
+If the Dataverse Installation supports multiple languages, the curation label translations should be added to the Bundle property files.
+Since the Curation labels are free text, while creating the key, it has to be converted to lowercase, replace space with underscore.
+
+Example:
+standard_process=Standard Process
+author_contacted=Author contacted
```

Comment:
If run `make html` you'll see that all three lines end up on a single line like this: Example: standard_process=Standard Process author_contacted=Author contacted

Perhaps you could put the key/value pairs in a code block like this:

```suggestion
Example::

  standard_process=Standard Process
  author_contacted=Author contacted
```